File: dataset/human/java/file_91.java
import java.util.Arrays;
import java.util.Scanner;

class Main{
	public static void main(String[] args){
		Scanner sc = new Scanner(System.in);
		while(sc.hasNext()){
			int N = sc.nextInt();
			long[] A = new long[N];
			int minus = 0;
			for(int i = 0; i < N; i++) {
				A[i] = sc.nextInt();
				if(A[i] < 0) {
					minus++;
					A[i] = -A[i];
				}
			}
			long ans = 0;
			Arrays.sort(A);
			for(int i = 0; i < N; i++) {
				ans += A[i];
			}
			if(minus % 2 == 0) System.out.println(ans);
			else System.out.println(ans - 2 * A[0]);
		}
	}
}

File: dataset/human/python/file_91.py
def xx(a,b,c):
    ans=0
    if c%a == 0:
        ans += 1
    while c>0:
        if c%b == 0:
            ans+=1
        c -= a
    return ans


r,g,b,n = map(int,input().split())
ans = 0
if n%r==0:
    ans += 1
while n>0:
    ans += xx(g,b,n)
    n -= r
print(ans)
